### DISCUSSION TITLE
lmtp-trusted-ips / support subnet or hostname

### DISCUSSION THREAD

**Discussion by ngocnha** [stalwartlabs/jmap-server#40] [Ideas]
Can we add support for configuring 'lmtp-trusted-ips' with a subnet (e.g., 10.89.0.0/24) or a hostname (e.g., smtp.myserver.com)? This would be beneficial as Docker containers always have a fixed hostname and subnet, but not a fixed IP address. It would provide us with more flexibility and make things easier.

**Comment by mdecimus** [comment]
Sure, I'll add that to the roadmap.
